app.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+require("express-async-errors");
+
+const app = express();
+
+app.set("view engine", "ejs");
+app.use(require("body-parser").urlencoded({ extended: true }));
+
+require("dotenv").config();
+const session = require("express-session");
+const MongoDBStore = require("connect-mongodb-session")(session);
+const url = process.env.MONGO_URI;
+
+const store = new MongoDBStore({
+    uri: url,
+    collection: "mySessions",
+});
+store.on("error", function (error) {
+    console.log(error);
+});
+
+const sessionParms = {
+    secret: process.env.SESSION_SECRET,
+    resave: true,
+    saveUninitialized: true,
+    store: store,
+    cookie: { secure: false, sameSite: "strict" },
+};
+
+if (app.get("env") === "production") {
+    app.set("trust proxy", 1);
+     sessionParms.cookie.secure = true; 
+}
+
+app.use(session(sessionParms));
+
+app.use(require("connect-flash")());
+
+app.get("/secretWord", (req, res) => {
+    if (!req.session.secretWord) {
+        req.session.secretWord = "syzygy";
+    }
+    res.locals.info = req.flash("info");
+    res.locals.errors = req.flash("error");
+    res.render("secretWord", {
+        secretWord,
+        errors: flash("error"),
+        info: flash("info"),
```

Comment:
I'm also getting the error `flash is not defined`. Did you mean to use `req.flash`?

```suggestion
        errors: req.flash("error"),
        info: req.flash("info"),
```